esp/services/ws_cloud/ws_cloudService.cpp

```diff
@@ -30,22 +30,35 @@ void CWsCloudEx::init(IPropertyTree* cfg, const char* process, const char* servi
         throw makeStringException(-1, "Can't initialize CWsCloudEx. The cfg is NULL.");
 
     VStringBuffer xpath("Software/EspProcess[@name=\"%s\"]/EspService[@name=\"%s\"]/PODInfoCacheSeconds", process, service);
-    unsigned podInfoCacheSeconds = cfg->getPropInt(xpath.str(), defaultPODInfoCacheForceBuildSecond);
+    unsigned k8sResourcesInfoCacheSeconds  = cfg->getPropInt(xpath.str(), defaultK8sResourcesInfoCacheForceBuildSecond);
     xpath.setf("Software/EspProcess[@name=\"%s\"]/EspService[@name=\"%s\"]/PODInfoCacheAutoRebuildSeconds", process, service);
-    unsigned podInfoCacheAutoRebuildSeconds = cfg->getPropInt(xpath.str(), defaultPODInfoCacheAutoRebuildSecond);
-    podInfoCacheReader.setown(new CPODInfoCacheReader("POD Reader", podInfoCacheAutoRebuildSeconds, podInfoCacheSeconds));
-    podInfoCacheReader->init();
+    unsigned k8sResourcesInfoCacheAutoRebuildSeconds = cfg->getPropInt(xpath.str(), defaultK8sResourcesInfoCacheAutoRebuildSecond);
+    k8sResourcesInfoCacheReader.setown(new CK8sResourcesInfoCacheReader("K8s Resource Reader", k8sResourcesInfoCacheAutoRebuildSeconds, k8sResourcesInfoCacheSeconds));
+    k8sResourcesInfoCacheReader->init();
+}
+
+void CWsCloudEx::ensureSuperUser(IEspContext& context)
+{
+#ifdef _USE_OPENLDAP
```

Comment:
Should we make this a specific feature permission instead of LDAP superuser?  Not a huge fan of ldap specific checking.

Reply:
Or is there a good reason for relying on superuser status?

Reply:
I think we do not need to rely on superuser status. There are 2 methods in WsCloud now. The access to those 2 methods are already controlled by the CloudAccess:
```
ESPservice [auth_feature("CloudAccess:ACCESS"), version("1.01"), exceptions_inline("./smc_xslt/exceptions.xslt")] WsCloud
{
    ESPmethod [auth_feature("CloudAccess:READ")] GetPODs(GetPODsRequest, GetPODsResponse);
    ESPmethod [auth_feature("CloudAccess:READ"), min_ver("1.01")] GetServices(GetServicesRequest, GetServicesResponse);
};
```